src/conn/conn_layered.c

```diff
@@ -1416,7 +1416,7 @@ __layered_copy_ingest_table(WT_SESSION_IMPL *session, WT_LAYERED_TABLE_MANAGER_E
          * ingest table. */
         if (tombstone != NULL) {
             tombstone->txnid = tw.stop_txn;
-            tombstone->upd_start_ts = tw.stop_ts != WT_TS_MAX ? tw.stop_ts : tw.stop_prepare_ts;
+            tombstone->upd_start_ts = tw.stop_ts;
```

Comment:
At the tw level, we should still only set one value. So if it is prepared, we should not set this.

Reply:
ok